resources/build/build-utils.sh

```diff
@@ -32,43 +32,20 @@
 #
 SHLVL=0
 
-function die () {
-    # TODO: consolidate this with fail() from shellHelperFunctions.sh
-    echo
-    echo "$*"
-    echo
-    exit 1
-}
-
-function findRepositoryRoot() {
+function _builder_findRepoRoot() {
     # See https://stackoverflow.com/questions/59895/how-to-get-the-source-directory-of-a-bash-script-from-within-the-script-itself
     # None of the answers are 100% correct for cross-platform
     # On macOS, requires coreutils (`brew install coreutils`)
     local SCRIPT=$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")
-    KEYMAN_ROOT=$(dirname $(dirname $(dirname "$SCRIPT")))
-    readonly KEYMAN_ROOT
+    REPO_ROOT=$(dirname $(dirname $(dirname "$SCRIPT")))
+    readonly REPO_ROOT
 }
 
-# Used to build script-related build variables useful for referencing the calling script
-# and for prefixing builder_finish_action outputs in order to more clearly identify the calling
-# script.
-#
-# Assumes that `findRepositoryRoot` has already been called, a condition met later on
-# within this script.
-function _builder_setBuildScriptIdentifiers() {
-  if [ ! -z ${THIS_SCRIPT+x} ]; then
-    THIS_SCRIPT_PATH="$(dirname "$THIS_SCRIPT")"
-    readonly THIS_SCRIPT_PATH
-    THIS_SCRIPT_NAME="$(basename "$THIS_SCRIPT")"
-    readonly THIS_SCRIPT_NAME
-    # Leaves only the part of the path based upon KEYMAN_ROOT.
-    THIS_SCRIPT_IDENTIFIER=${THIS_SCRIPT_PATH#"$KEYMAN_ROOT/"}
-    readonly THIS_SCRIPT_IDENTIFIER
-  fi
-}
+# Source builder_script (how to move function _builder_findRepoRoot() to .inc.sh?)
+. "$REPO_ROOT/resources/build/builder.inc.sh"
```

Comment:
The intent of this PR is so help.keyman.com can source `builder.inc.sh` without needing `build-utils.sh`.

I would've preferred to also include `_builder_findRepoRoot()` inside of `builder.inc.sh`, but had a conundrum of needing `$REPO_ROOT` defined in order to source `builder.inc.sh`.